### PR TITLE
VSCode  Suggestions & DocBlock Comments Fixed

### PR DESCRIPTION
**Pull request by krishna-gujjjar**
Fix Menu Component invalid `@return` type, other PHP DocBlock issue & renaming properties based on `php-cs`. There are no changes on the source code beside renamed some classes private properties and DocBlock.

### CODE REVIEW

File: src/Menus/HasMenuIcons.php [hunk 1]
```diff
@@ -55,43 +78,55 @@ public function setIconUrl(string $url)
...
-     * Returns the full icon tag: either a <i> tag for FontAwesome
-     * icons, or an <img> tag for images.
+     * Returns HTML tag depends Menu Icon type `iconUrl` or `faIcon`
+     *
+     * Returns the full icon tag: either a `<i>` tag for FontAwesome icons,
+     * or an `<img>` tag for images.
+     *
+     * @param string $classOrIconUrl FontAwesome Icon Class Name or Icon URL
+     *
+     * @return string
```
Comment (neznaika0) [inline src/Menus/HasMenuIcons.php]: There is no need to duplicate callouts in PHPDoc if they are set in the method. a note to all such changes

File: src/Menus/HasMenuIcons.php [hunk 2]
```diff
@@ -7,45 +7,68 @@
...
+ * @package  Bonfire
+ * @author   Lonnie Ezell <lonnieje@gmail.com>
+ * @license  MIT https://opensource.org/licenses/MIT
+ * @link     https://github.com/lonnieezell/Bonfire2/
+ * @see      https://github.com/lonnieezell/Bonfire2/
  */
 
 namespace Bonfire\Menus;
 
+/**
```
Comment (neznaika0) [inline src/Menus/HasMenuIcons.php:20]: Why do we need duplicates in the description?

File: src/Menus/HasMenuIcons.php (hunk 1 above)
Comment (krishna-gujjjar) [reply src/Menus/HasMenuIcons.php]: Thanks for review @neznaika0 

This is done by a VSCode's plugin called [PHP Sniffer & Beautifier](https://marketplace.visualstudio.com/items?itemName=ValeryanM.vscode-phpsab). I will  remove those unnecessary comment and run `composer run style` first before commit. 

Please review it again and tell me if any updates needed.

File: src/Menus/HasMenuIcons.php (hunk 2 above)
Comment (krishna-gujjjar) [reply src/Menus/HasMenuIcons.php:20]: I removed duplicate File DocBlock in last commit.

**Review by neznaika0 — APPROVED** [review]
I'm not a Bonfire 2 developer, but my opinion is all right

**Comment by krishna-gujjjar** [comment]
Here are some screenshots after all the changes.

| Before | After |
|--------|-------|
| Screenshot 2023-08-06 at 11 36 35 AM  | Screenshot 2023-08-06 at 1 32 29 PM   |

Screenshot 2023-08-06 at 8 47 54 PM

✨ Cools Suggestion with inline example preview.

**Review by lonnieezell — APPROVED** [review]

**Pull request merged**